pontoon/base/static/js/tabs.js

```diff
@@ -0,0 +1,168 @@
+
+  // History
+  window.onpopstate = function(event) {
+    loadTabContent(window.location.pathname + window.location.search);
+  };
+
+  // Menu
+  $('body').on('click', '#middle .links a, #main .contributors .links a', function(e) {
+    // Keep default middle-, control- and command-click behaviour (open in new tab)
+    if (e.which === 2 || e.metaKey || e.ctrlKey) {
+      return;
+    }
+
+    e.preventDefault();
+
+    var url = $(this).attr('href');
+    loadTabContent(url);
+    window.history.pushState({}, '', url);
+  });
+
+  function showTabMessage(text) {
+    var message = $('<p>', {
+          class: 'no-results',
+          html: text
+        });
+
+    container.empty().append(message);
+  }
+
+  function updateTabCount(tab, count) {
+    tab.find('span').remove();
+    if (count > 0) {
+      $('<span>', {
+        class: 'count',
+        html: count
+      }).appendTo(tab);
+    }
+  }
+
+  function loadTabContent(path) {
+    if (inProgress) {
+      inProgress.abort();
+    }
+
+    var url = path.split(urlSplit)[1],
+        tab = $('#middle .links a[href="' + path.split('?')[0] + '"]');
+
+    // Update menu
+    $('#middle .links li').removeClass('active');
+    tab.parents('li').addClass('active');
+
+    if (url !== '/bugs/') {
+      inProgress = $.ajax({
+        url: '/' + urlSplit + '/ajax' + url,
+        success: function(data) {
+          container.empty().append(data);
+
+          if (url.startsWith('/contributors/')) {
+            var count = $('table > tbody > tr').length;
+            updateTabCount(tab, count);
+          }
+
+          if (url === '/') {
+            $('.controls input').focus();
+          }
+        },
+        error: function(error) {
+          if (error.status === 0 && error.statusText !== "abort") {
+            showTabMessage('Oops, something went wrong.');
+          }
+        }
+      });
+
+    } else {
+      var locale = $('#server').data('locale');
+      inProgress = $.ajax({
+        url: 'https://bugzilla.mozilla.org/rest/bug',
+        data: {
+          'field0-0-0': 'component',
+          'type0-0-0': 'regexp',
+          'value0-0-0': '^' + locale + ' / ',
+          'field0-0-1': 'cf_locale',
+          'type0-0-1': 'regexp',
+          'value0-0-1': '^' + locale + ' / ',
+          'resolution': '---',
+          'include_fields': 'id,summary,creation_time,last_change_time'
+        },
+        success: function(data) {
+          if (data.bugs.length) {
+            data.bugs.sort(function(l, r) {
+              return l.last_change_time < r.last_change_time ? 1 : -1;
+            });
+
+            var tbody = $('<tbody>'),
+                formatter = new Intl.DateTimeFormat('en-GB', {
+                  day: 'numeric',
+                  month: 'long',
+                  year: 'numeric'
+                });
+
+            $.each(data.bugs, function(i, bug) {
+              tr = $('<tr>', {
+                title: bug.summary
```

Comment:
I simulated a situation when bugzilla returns summary which contains script tags and I successfully displayed an js alert. From what I checked, bugzilla returns raw contents of the summary in the api (e.g. https://bugzilla.mozilla.org/rest/bug/344890). So, If I'm not wrong, then doNotRender() would be advised here.